lib/testing_api_onlyoffice_com/test_instance/main_page/document_builder_api/left_side_navigation.rb

```diff
@@ -0,0 +1,38 @@
+require 'httparty'
+require 'onlyoffice_file_helper'
+module TestingApiOnlyfficeCom
+  # https://user-images.githubusercontent.com/18173785/37905775-9964ebb6-3108-11e8-8f98-480cbb1c2906.png
+  # /docbuilder/basic
+  module LeftSideNavigation
+    include PageObject
+
+    # actions
+    link(:introduction, xpath: '//a[@class="selected"]')
```

Comment:
On https://api.onlyoffice.com/docbuilder/gettingstarted this page this xpath will be not for `introduction` part